### PR TITLE
turtle: Capitalise 'PhotoImage'

### PR DESCRIPTION
**Pull request by Son-Dongwoo**
The word "Photoimage" seems a typo. This PR corrects the typo to "PhotoImage".

### CODE REVIEW

File: Misc/NEWS.d/next/Library/2023-09-06-06-06-43.gh-issue-108958.vG6m73.rst [hunk 1]
```diff
@@ -0,0 +1 @@
+The word "Photoimage" seems a typo. This PR corrects the typo to "PhotoImage".
```
Comment (corona10) [inline Misc/NEWS.d/next/Library/2023-09-06-06-06-43.gh-issue-108958.vG6m73.rst]: Please remove NEWS.d, it is not necessary for simple typo.

**Review by corona10 — DISMISSED** [review]
See: https://github.com/python/cpython/pull/108958#discussion_r1329867344

**Review by AA-Turner — APPROVED** [review]
Seems this is a Tk ref: https://docs.python.org/3/library/tkinter.html#images

A

File: Misc/NEWS.d/next/Library/2023-09-06-06-06-43.gh-issue-108958.vG6m73.rst (hunk 1 above)
Comment (Son-Dongwoo) [reply Misc/NEWS.d/next/Library/2023-09-06-06-06-43.gh-issue-108958.vG6m73.rst]: Yes, thank you

**Comment by dnd-qodqks** [comment]
@AA-Turner Thanks for the fix. :+1:

**Review by corona10 — APPROVED** [review]

**Comment by dnd-qodqks** [comment]
Thank you for the merge!

**Pull request merged**